runner/operator/access/v1_0_0/structural_variants/__init__.py

```diff
@@ -15,7 +15,7 @@
 TUMOR_OR_NORMAL_SEARCH = '-L0'
 SAMPLE_ID_SEP = '_cl_aln'
 ACCESS_DEFAULT_SV_NORMAL_ID = 'DONOR22-TP'
-ACCESS_DEFAULT_SV_NORMAL_FILENAME = 'DONOR22-TP_cl_aln_srt_MD_IR_FX_BR.bam'
+ACCESS_DEFAULT_SV_NORMAL_FILENAME = 'DONOR22-TP_cl_aln_srt_MD_IR_FX_BR.bam$'
```

Comment:
does this have to be regex, prepended with `r` like the others?

Reply:
I didn't actually have the issue with this file but yes I've made it a regex as well to be sure